packages/stylis/src/index.js

```diff
@@ -1,2 +1,1215 @@
-// @flow
-export { default } from './stylis.min'
+/* eslint-disable */
```

Comment:
Could you try running this through closure compiler and see if it has any size wins?

Reply:
I've actually already tried this, the output for ESM build was slightly higher, UMD had like 100b in favour of closure.

I've also compared output and the main difference (for ESM) was:
- closure have inlined "enum" strings
- closure have inlined 2 small helpers responsible for reading string and array lengths

Other than that there was just some changes to the minified code, slightly different structure of loops etc but nothing substantial really.

Reply:
cc @thysultan - this might slightly interest you, not sure if you have seen this comment and my "findings"

also - an additional note as I've just remembered one more thing observed, the closure compiler was able to minify 2 properties of the node structure (dont recall which ones), which is not correct from our perspective as node's shape is stylis public API, this probably could be prevented somehow, but im not too familiar with the closure compiler, so this would require some research on my part

but as mentioned - i haven't really observed any substantial changes between terser and closure compiler outputs, so not sure if it's worth looking into using it. Keep in mind that I've really looked quickly through those outputs and haven't done extensive analysis